dockerfiles/init/modules/nginx/templates/http_nginx.conf.erb

```diff
@@ -62,8 +62,7 @@ error_log                     /var/log/nginx/error.log error;
         #gzip_proxied expired no-cache no-store private auth;
         #gzip_types text/plain text/css text/xml text/javascript application/x-javascript application/xml;
 
-        resolver                    127.0.0.1 8.8.8.8 ipv6=off;
-
+        resolver                    <% if @dns_resolvers != 'NULL' -%>8.8.8.8<% else -%><%= @dns_resolvers.gsub(',', ' ') %><% end -%> ipv6=off;
```

Comment:
if statement look strange maybe I was confused you by asking check != null
but seems in that case it should looks like
`<% if @dns_resolvers == 'NULL' -%>8.8.8.8<% else -%><%= @dns_resolvers.gsub(',', ' ') %><% end -%>`
otherwise 8.8.8.8 will never be by defaul